tl/features/cell_context_matches.py

```diff
@@ -273,11 +283,11 @@ def initialize(self, raw_input_df, context_dict, label_column):
                                            score=context_result['score'],
                                            best_match=context_result['best_match']
                                            )
+        self.process(row_column_label_dict)
 
-    def process(self):
+    def process(self, row_column_label_dict: dict):
```

Comment:
row_column_label_dict is a class variable, should not be passed. remove this parameter